src/lib/use-resource.js

```diff
@@ -53,12 +53,3 @@ export function useResource(fn, deps) {
 	else if (state.status === 'error') throw state.result;
 	throw state.promise;
 }
-
-export function FakeSuspense(props) {
-	this.__c = childDidSuspend;
-	return this.state && this.state.suspended ? props.fallback : props.children;
-}
-
-function childDidSuspend(err) {
-	err.then(() => this.forceUpdate());
-}
```

Comment:
Not necessarily a tearing issue, but something that needed to be fixed. `this.state.suspended` isn't ever truthy, and so the fallback doesn't get rendered. Most obviously seen on the home page, that first code block is gets emptied (and so collapses down) then repopulates once prism runs locally. Pretty jarring. 

Tbh, I rather mindlessly copied this from #793 and didn't look too closely at it. Swapping it out for standard `Suspense` from `preact/compat` seems reasonable? Dunno if there's something I'm missing though.